account-kit/rn-signer/ios/implementation/KeychainHelper.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+import Security
+import CryptoKit
+
+let account_kit_ephemeral_private_key = "ephemeralPrivateKey"
+
+// Keep save_ephemeral_private_key_query mutable
+var save_ephemeral_private_key_query: [String: Any] = [
+    kSecClass as String: kSecClassGenericPassword,
+    kSecAttrAccount as String: account_kit_ephemeral_private_key,
+    kSecAttrAccessible as String: kSecAttrAccessibleAfterFirstUnlock
+]
+
+let get_ephemeral_private_key_query: [String: Any] = [
+    kSecClass as String: kSecClassGenericPassword,
+    kSecAttrAccount as String: account_kit_ephemeral_private_key,
+    kSecReturnData as String: true,
+    kSecMatchLimit as String: kSecMatchLimitOne
+]
+
+let delete_ephemeral_private_key_query: [String: Any] = [
+    kSecClass as String: kSecClassGenericPassword,
+    kSecAttrAccount as String: account_kit_ephemeral_private_key
+]
+
+
+class KeychainHelper {
+    static let shared = KeychainHelper() // Singleton instance
+
+    private init() {} // Prevent external instantiation
+
+    func savePrivateKeyToKeychain(_ privateKey: P256.KeyAgreement.PrivateKey) {
+        let keyData = privateKey.rawRepresentation
+        save_ephemeral_private_key_query.updateValue(keyData, forKey: kSecValueData as String)
```

Comment:
Arguably if we wanted we could just make generating this a function of this class and that'll simplify needing to update the value and maintaining immutability. Updating a global variable seems like a bad idea.